tests/integration/kubernetes/gha-run.sh

```diff
@@ -554,11 +554,18 @@ function cleanup_nydus_snapshotter() {
 	else
 		kubectl_retry delete --ignore-not-found -f "misc/snapshotter/base/nydus-snapshotter.yaml"
 	fi
-	sleep 180s
+
+	# The extensive sleep durations lead to certain tests timing out. 
+	# Commenting the sleep for archival purposes for now.
+	#sleep 180s
+
```

Comment:
I think we can delete the comments - if the sleeps aren't needed? Thanks

Reply:
done!